src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
+    private static final int ZERO_INDEX = 0;
+    private static final int ONE_INDEX = 1;
+    private int supply = 0;
+    private int buy = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = calculateData(readFile(fromFileName));
+        writeResult(toFileName, report);
+
+    }
+
+    public String readFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File not found ", e);
```

Comment:
```suggestion
            throw new RuntimeException("File not found: " + fromFileName, e);
```

make exceptions message more informative